tests/Umbraco.Tests.AcceptanceTest/playwright.config.ts

```diff
@@ -23,7 +23,8 @@ export default defineConfig({
   // We don't want to run parallel, as tests might differ in state
   workers: 1,
   /* Reporter to use. See https://playwright.dev/docs/test-reporters */
-  reporter: process.env.CI ? 'line' : 'html',
+  //reporter: process.env.CI ? 'line' : 'html',
+  reporter: [['junit', { outputFile: 'results/results.xml' }]],
```

Comment:
```suggestion
  reporter: [['line'], ['junit', {outputFile: 'results/results.xml'}]],
```
We should still have the line for the overview in the pipeline run. Which looks like this with only the junit 
![image](https://github.com/user-attachments/assets/fdd3cf7f-b435-4694-b0be-f0d5f6017726)